sites/all/modules/reol_base/reol_base.field.inc

```diff
@@ -139,6 +145,20 @@ function reol_base_field_formatter_view($entity_type, $entity, $field, $instance
           );
         }
         break;
+
+      case 'reol_base_series':
+        $data = reol_base_get_series_data($entity);
+        if (NULL !== $data) {
+          // This is almost as ugly as the stuff in ting_field_formatter_view.
+          $element[$delta] = array(
+            '#markup' => '<span class="material__series-icon"><svg width="13" height="14" viewBox="0 0 13 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would say that this requires a theme function... please.

Reply:
I've cleaned this up and moved the icon to css. Using a theme function would be overkill.